x-pack/plugin/sql/qa/server/src/main/resources/version.csv-spec

```diff
@@ -0,0 +1,206 @@
+// To mute tests follow example in file: example.csv-spec
```

Comment:
I suggest adding more tests with queries with:
- `HAVING`
- `GROUP BY` and `COUNT`
- `NULLS FIRST`
- other functions using `version`, including in `ORDER BY`. For example `SELECT CASE WHEN LEFT(version, 1) = '1' THEN 1 WHEN LEFT(version, 1) = '2' THEN 2 ELSE 0 AS v, COUNT(*) GROUP BY 1 ORDER BY 1 DESC`.
- `ORDER BY` with `DESC`
- `SELECT version, version as v, version as v2 GROUP BY 1 ORDER BY 2 LIMIT 3`

Reply:
HAVING unfortunately is supported only for a limited set of data types, and Version is not one of them.